### ISSUE TITLE
Sending command to the server hang if there is no response.

### ISSUE THREAD

**Issue by Br31zh** [gekware/minecraft-server-hibernation#158]
### Current Behavior (🐛 Bug)

When sending command to the Minecraft server by writing on MSH’s standard input, if there is no response by the server in the console, it’s impossible to send another command (even to msh it seems) until another action is done (like a player joining/quitting the game).

### Expected Behavior

Sending a command without output don’t block new commands. Maybe adding a timeout, or don’t wait for output for manually send commands (I think that MSH don’t process them anyway, right?).

### Steps to Reproduce

Send a command that produce no console output (in my case it’s from a mod) from MSH’s standard input, then try to send other commands.

### MSH Logs

```
mine say test
2022/08/15 12:46:06 [serv  ≡≡  ] ms command: say test	(origin: user input)
2022/08/15 12:46:06 [serv  ≡≡  ] [12:46:06] [Server thread/INFO]: [Not Secure] [Server] test
2022/08/15 12:46:06 [serv  ≡≡  ] a chat message was sent
mine dynmap sendtoweb test
2022/08/15 12:46:22 [serv  ≡≡  ] ms command: dynmap sendtoweb test	(origin: user input)
mine say foo
mine say bar
2022/08/15 12:46:42 [serv  ≡≡  ] [12:46:42] [Server thread/INFO]: Br31zh lost connection: Disconnected
2022/08/15 12:46:42 [serv  ≡≡  ] A PLAYER LEFT THE SERVER! - 0 players online
2022/08/15 12:46:42 [serv  ≡≡  ] ms command: say foo	(origin: user input)
2022/08/15 12:46:42 [serv  ≡≡  ] [12:46:42] [Server thread/INFO]: Br31zh left the game
2022/08/15 12:46:42 [serv  ≡≡  ] ms command: say bar	(origin: user input)
2022/08/15 12:46:42 [serv  ≡≡  ] [12:46:42] [Server thread/INFO]: [Not Secure] [Server] foo
2022/08/15 12:46:42 [serv  ≡≡  ] a chat message was sent
2022/08/15 12:46:42 [serv  ≡≡  ] [12:46:42] [Server thread/INFO]: [Not Secure] [Server] bar
2022/08/15 12:46:42 [serv  ≡≡  ] a chat message was sent
```

### MSH-Version

2.4.7

### Operating system

Arch Linux

### Minecraft Version

1.19.2

### Minecraft Server Software

Tested with Fabric and Quilt.

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details.
(Mentionned in #51 )

**Comment by gekigek99** [comment]
Do you have a vanilla command that reproduces this behaviour?

_To avoid having to test with mods which will complicate the issue_

**Comment by gekigek99** [comment]
fixed in d37135659a8849c0bd6160654ab2d1be8291f120

compile the dev branch and tell me if you confirm, so I can close the issue.

_if you use msh and want to support it please give a star and consider becoming a sponsor_

**Comment by gekigek99** [comment]
thanks for reporting btw <3
